Sources/ShapedArray/Operators/Basic.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Basic.swift
+//  
+//
+//  Created by Khalid Alotaibi on 12/26/22.
+//
```

Comment:
I'll add this to the readme, but we need to follow the same copywrite as in the other files. E.g. https://github.com/swift-numerics-workgroup/swift-shaped-array/blob/99c2f10036a42f66bead022f35d1030d1d39d677/Sources/ShapedArray/ShapedArray.swift#L1-L14